src/components/AnnotationShareControl.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  CopyIcon,
+  IconButton,
+  Input,
+  InputGroup,
+  Popover,
+  ShareIcon,
+} from '@hypothesis/frontend-shared';
+import { useCallback, useEffect, useRef, useState } from 'preact/hooks';
+
+import type { Group } from '../helpers';
+import type { Annotation } from '../helpers/annotation-metadata';
+import { isPrivate } from '../helpers/permissions';
+import { isIOS } from '../utils/user-agent';
+
+export type AnnotationShareControlProps = {
+  /** The annotation in question */
```

Comment:
```suggestion
  /** The annotation in question */
```

This JSDoc doesn't add value, I would just remove it.